extensions/wikia/ExactTargetUpdates/tests/client/ExactTargetClientTest.php

```diff
@@ -202,6 +202,10 @@ public function testDefaultClientConstruction() {
 			return $getExactTargetClient->invoke( $client );
 		} );
 
+		# normalize resources to allow assertEquals to pass in PHP7
+		$expected->sdl = is_resource( $expected->sdl );
+		$soapClient->sdl = is_resource( $soapClient->sdl );
```

Comment:
👍 LG, tej - although this code is going to be retired completely in a matter of days.